indexer.py

```diff
@@ -82,6 +89,45 @@ def calc_tvl(self, session, indx, block):
 
         StatsTVL.insert(session, block, timestamp, tvl)
 
+    def calc_tvc(self, session, indx, block):
+        # read list of protocols from local csv file
+        protocols = {}
+        with open("./meta/protocols.csv", newline="") as csv_file:
```

Comment:
Any changes to the `protocols.csv` can cause issues during runtime.
Maybe we can read from the csv in settings.py and parse to dictionary and use the in memory object here.

Reply:
yeah, good call